src/sampling/homogeneous.jl

```diff
@@ -40,6 +40,20 @@ function sample(rng::AbstractRNG, triangle::Triangle{Dim,T}, method::Homogeneous
   (randpoint() for _ in 1:(method.size))
 end
 
+function sample(rng::AbstractRNG, ball::Ball{Dim,T}, method::HomogeneousSampling) where {Dim,T}
```

Comment:
Can you please reimplement this in terms of the new parametric representation of Ball? In the master branch you can ball(u, v) or ball(u, v, w) to get a point in the ball using polar or spherical coordinates.

Reply:
Sure, I will do that.

Reply:
Thank you! We need some tests after that 👌🏽